### PR TITLE
Reorganise files into classes (up to Run_Categorisation)

### PR DESCRIPTION
**Pull request by harvard-tham**

### CODE REVIEW

**Review by lixitrixi — APPROVED** [review]
New organization looks good: "Old Classes" will act as an archive of the old version and we can copy code from there as we need it.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-These `ARTwarp.m` files have been run successfully using Matlab 2016a and 2016b
-
-`ARTwarp.m` will open whistle contours extracted using Beluga
-
-`ARTwarp_txt.m` provides the option to open `.ctr` (Beluga), `.csv`, or `.txt` files
-
-This version of `ARTwarp_txt` displays the iteration number and number of whistles 
-analysed in the command window while ARTwarp is running. It saves the results at 
-the end of each iteration.
+# ARTwarp
```
Comment (olexandr-konovalov) [inline README.md:1]: This looks strange - it should not show all differences in lines 1-16 which are untouched. Which editor and which operating system do you use? How do you make commits?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
+`ARTwarp.m` will open whistle contours extracted using Beluga
+
+`ARTwarp_txt.m` provides the option to open `.ctr` (Beluga), `.csv`, or `.txt` files
+
+This version of `ARTwarp_txt` displays the iteration number and number of whistles 
+analysed in the command window while ARTwarp is running. It saves the results at 
+the end of each iteration.
+
+
+## Old Classes
```
Comment (olexandr-konovalov) [inline README.md:19]: I wonder if this is the right way to do this. Maybe you should have started another branch instead? Michael's code will be there under version control anyway at least for a while, until we decide to keep that  branch or delete it)

**Comment by olexandr-konovalov** [comment]
> New organization looks good: "Old Classes" will act as an archive of the old version and we can copy code from there as we need it.
@lixitrixi sounds like replacing version control by shuffling files around - I am not convinced that this is what we needed.

Sometimes I am visited by an idea that we need to create a repository artwarp2 and work on a new version there from scratch. I haven't seen this coming last year, but now, when you completely change the architecture, that may be more viable...

**Review by olexandr-konovalov — COMMENTED** [review]
@harvard-tham can you figure out how to add some unit or script test? I want that to be able to use some actual tests with https://github.com/dolphin-acoustics-vip/artwarp/pull/43

**Comment by lixitrixi** [comment]
> > New organization looks good: "Old Classes" will act as an archive of the old version and we can copy code from there as we need it.
> > @lixitrixi sounds like replacing version control by shuffling files around - I am not convinced that this is what we needed.
> 
> Sometimes I am visited by an idea that we need to create a repository artwarp2 and work on a new version there from scratch. I haven't seen this coming last year, but now, when you completely change the architecture, that may be more viable...

Now that you suggest it... I think this would be a good course of action.

**Comment by olexandr-konovalov** [comment]
@lixitrixi now, after we are done with #26, perhaps we don't need a new repository after all! This way, we can publicise https://github.com/dolphin-acoustics-vip/artwarp/wiki as ARTWarp website and the URL seems hopefully to be stable enough for now.

The plan is that we have now a `stable-1` branch with ARTwarp v1.0 released from it. In case of any minor fixes needed, they should be made there, and we can release from that branch v1.1, v1.2 etc.

In the meantime, we work in the main branch here on ARTwarp v2.0. As soon as it stabilises, we create `stable-2` branch and from there will be making v2.0, v2.1 etc., and continuing to work in `main` on what could become eventually v3.0 in the same way.

Therefore, for new developments it will be simpler, as you don't have to keep all work in the 'rewrite' branch.

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]
Can't be merged in the present form. @harvard-tham and me will discuss tomorrow.

**Review by olexandr-konovalov — APPROVED** [review]

**Pull request merged**